march_hardware/include/march_hardware/TemperatureGES.h

```diff
@@ -11,14 +11,14 @@ namespace march4cpp
 class TemperatureGES : public Slave, TemperatureSensor
 {
 private:
-  int byteOffset;
+  int temperatureByteOffset;
```

Comment:
Why rename it when it is part of a class with Temperature in its name?

Reply:
the GES may be a temperatureGES, but it may in the future return multiple variables, e.g. error codes, variances etc. It is then important that this byteoffset variable refers to the actual temperature.

Also in case this class is refactored to something more general, e.g. 'GES', this is necessary.